app/styles/ember-frost-table/_frost-table.scss

```diff
@@ -3,12 +3,27 @@
 //
 
 .frost-table {
-  table-layout: fixed;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+
+  &-body-scroll {
+    display: flex;
+    flex: auto;
```

Comment:
The growth here makes sense, but I don't know if the scroll needs to be a flex container itself (i.e. `display: flex`)